### ISSUE TITLE
Editing item categories triggers a LUA error: unknown method `CategoriesChanged`

### ISSUE THREAD

**Issue by Elberet** [doadin/Baggins#118]
Changing an existing category's associated filtering rules triggers this LUA error:

```
Baggins/Baggins-Filtering.lua:409: attempt to call method 'CategoriesChanged' (a nil value)
[string "@Baggins/Baggins-Filtering.lua"]:409: in function `OnRuleChanged'
```

This was recorded on Cataclysm Classic with an english language client.
Current Baggins version is 5.4.1, last known good version is 5.3.3

**Comment by doadin** [comment]
This should be fixed via this commit: c64e8b51a06e94ef915cccc752dcbbc2fec05a33 , included in release build v5.4.2 .

**Comment by doadin** [comment]
Looking at the other issue now as well, will update there.
